pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/kit/ClinicalKitDto.java

```diff
@@ -135,6 +135,7 @@ private String getParticipantGender(ElasticSearchParticipantDto participantBySho
                 return oncHistoryWithGender.get().getGender();
             }
         }
+        logger.info("Participant {} did not have gender on tissue pages, will look into activities", participantByShortId.getParticipantId());
```

Comment:
We need to check that the `ElasticSearchParticipantDto` has a participant ID before using it (starting at line 129 above). I suspect a missing participant ID is the cause of this particular bug.
Likewise for getting activities, unless `getGenderFromActivities` is robust for that already.